src/test/scala/org/ergoplatform/nodeView/viewholder/ErgoNodeViewHolderSpec.scala

```diff
@@ -108,6 +108,7 @@ class ErgoNodeViewHolderSpec extends ErgoPropertyTest with NodeViewTestOps with
     }
   }
 
+  // todo: rework tests
```

Comment:
specify what do you mean here

Reply:
The fact that blocks could be applied only sequentially contradicts with original logic of this test, so it should be reimplemented from scratch.